server/cmwell-ws/app/wsutil/Encoders.scala

```diff
@@ -432,21 +432,21 @@ object Encoders {
     }
   }
 
-  implicit object SearchResultsEncoder extends Format[SearchResults] {
-    def reads(json: JsValue): JsResult[SearchResults] = ???
-
-    def writes(o: SearchResults): JsValue = {
-      val structure = new FieldAggregator
-      structure += "type" -> Json.toJson("SearchResults")
-      structure += "fromDate" -> Json.toJson(o.fromDate)
-      structure += "toDate" -> Json.toJson(o.toDate)
-      structure += "total" -> Json.toJson(o.total)
-      structure += "offset" -> Json.toJson(o.offset)
-      structure += "length" -> Json.toJson(o.length)
-      structure += "infotons" -> Json.toJson(o.infotons.map(Json.toJson(_)))
-      structure.get()
-    }
-  }
+//  implicit object SearchResultsEncoder extends Format[SearchResults] {
```

Comment:
is there a reason to keep this code commented out? (also below again, and in the next file)